src/index.js

```diff
@@ -2,10 +2,13 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
+import axios from 'axios';
 
 import store from './redux/configureStore';
 import App from './App';
 
+axios.defaults.baseURL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/eZNPE2dPLtWHMUhnsTBH/books';
```

Comment:
- [ ] Please 🙏🏽 refactor your code and use the API URL in the corresponding redux file to have a clean code.